verifyincluded.go

```diff
@@ -31,31 +33,17 @@ func proofPath(proof [][]byte) string {
 }
 
 func verifyEvent(
-	event *logverification.VerifiableEvent, massifHeight uint8, massifReader MassifReader,
-	forTenant string,
-	tenantLogPath string,
+	event *app.AppEntry, logTenant string, mmrEntry []byte, massifHeight uint8, massifGetter MassifGetter,
```

Comment:
`mmrEntry` also exists on the `AppEntry` type right? Why is it passed separately here?

Reply:
Good question!

I have left the `verifyEvent` function here, as its a low level example of how to verify the inclusion of an event, without the log verification module.

I believe that was an important factor before in decision making, happy to take it out because its not used anywhere, but if we want an example code in veracity of the fastest way to verify and event using low level code, this is it

Reply:
added a comment to the function saying its an example function

Reply:
in essence its redundant code, and i would be more than happy to get rid of it. 

Although i remember it was quite a sticking point in the past. If at a later date we decide we don't need it, it is self contained and we can just delete?